### ISSUE TITLE
Error while setting the Secret-Store with the credentials.

### ISSUE THREAD

**Issue by sudharsank** [pnp/powershell#2430]
### Reporting an Issue or Missing Feature
There is an issue when I tried to use the feature **Authenticating with pre-stored credentials using the Secrets Management Module from Microsoft (Multi-Platform)**. 

### Expected behavior 
Install-Module -Name Microsoft.PowerShell.SecretManagement -AllowPrerelease
Install-Module -Name Microsoft.PowerShell.SecretStore -AllowPrerelease
Set-SecretStoreConfiguration
Set-Secret -Name "yourlabel" -Secret (Get-Credential)

After following the above steps the **Secret Store** should be updated with the credentials.

### Actual behavior 
There is an error when trying to **Set-Secret**. Below is the error message received.
```
WARNING:
        There are currently no extension vaults registered.
        At least one vault must be registered before SecretManagement can add or retrieve secrets.
        You can download SecretManagement extension vault modules from PowerShellGallery.
        https://aka.ms/SecretManagementVaults
```

### Steps to reproduce behavior
I just followed the steps mentioned in the document.

### What is the version of the Cmdlet module you are running?
1.11.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by sudharsank** [comment]
Hi,
I just followed the template to raise an issue. Actually I just fixed the issue following the steps below
Before running the **Set-Secret** command, the below command has to be ran to register the secret store
```
Register-SecretVault -Name <SecretStore> -ModuleName Microsoft.PowerShell.SecretStore -DefaultVault -AllowClobber
```
Below link is very much helpful in resolving the issue. I hope the above command and the link can be updated in the document.
[SecretManagement and SecretStore Release Candidate 2](https://devblogs.microsoft.com/powershell/secretmanagement-and-secretstore-release-candidate-2/)

Thanks
